libcxx/test/std/containers/container.adaptors/flat.map/flat.map.cons/deduct.pass.cpp

```diff
@@ -31,19 +31,19 @@ using PC = std::pair<const int, long>;
 
 void test_copy() {
   {
-    std::flat_map<long, short> source = {{1, 2}, {2, 3}};
+    std::flat_map<long, short> source = {{1, static_cast<short>(2)}, {2, static_cast<short>(3)}};
```

Comment:
IMO this does not improve the readability of the code. Do you have an example of the MSVC diagnostic?

Reply:
Now I'm switching to `short{N}` for short.